modules/ROOT/pages/https-reference.adoc

```diff
@@ -2,9 +2,33 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
+:keywords: configure, HTTPS endpoint
+
+You automatically deploy an API proxy using an HTTP listener, inbound HTTP endpoint, with minimal setup. You can use this approach if you do not foresee any security concerns in your Anypoint Platform API setup. Otherwise, you must configure your API proxy with an HTTPs listener (inbound HTTPS endpoint), instead of an HTTP listener.
+
+MuleSoft requires that you provide your own certificate when configuring your HTTPs setup. The certificate you use can be a self-signed certificate to be used for demo purposes, custom TLS certificate with validity within your organization, or a trusted Certificate Authority (CA) certificate.
+
+If your backend uses HTTPs with a trusted CA certificate, you do not need to perform any special action to communicate with it from the API proxy. If your backend uses HTTPs with a self-signed certificate or custom TLS certificate, you must configure the API proxy requester with the corresponding certificate.
+
+Depending on your business needs, you can use any of the following recommended ways to configure your API Proxy to use HTTPs:
+
+* xref:building-https-proxy.adoc[HTTPs configured through the UI by leveraging Secret Manager]:
++ 
+The certificate is uploaded to a secure vault within API Platform, and then made available when configuring the proxy through the Secret Manager. Once configured, you can deploy the secure API pProxy to the target runtime, such as CloudHub, Runtime Fabric, and on-premises.
+* xref:proxy-domain-support.adoc[HTTPs defined in already deployed Mule Domain]
```

Comment:
"HTTPs defined in already deployed Mule Domain" -> "Certificate defined in already deployed Mule Domain"

Reply:
Done! Thanks!